<?xml version="1.0" encoding="utf-8"?>
<launch>
	<!-- declare arg to be passed in -->
	<arg name="data_file" default="$(find cpt_matching_algorithms)/../../smb_2020-07-01-13-30-10_frame_turned_down.bag" />
	<arg name="save_target" default="$(find cpt_matching_algorithms)/../../data_set_arche_13_30_10" />
	<arg name="topic_of_lidar_in_bag" default="/rslidar_points" />

    <node name="data_to_pcl_reader" pkg="cpt_matching_algorithms" type="data_to_pcl_reader" output="screen">
    	<param name="raw_data" value="$(arg data_file)" />
        <param name="save_folder" value="$(arg save_target)" />
        <param name="point_topic" value="$(arg topic_of_lidar_in_bag)" />
        <param name="gt_topic" value="/leica/position" />
        <param name="add_ground_truth" value="true" />
        <param name="start_idx" value="0" />
    </node>
</launch>
